src/java.base/share/classes/java/net/InetAddress.java

```diff
@@ -814,7 +822,7 @@ public String getCanonicalHostName() {
     }
 
     /**
-     * Returns the hostname for this address.
+     * Returns the fully qualified domain for this address.
```

Comment:
Both `getHostName` and `getCanonicalHostName` talk about `fully qualified domain name`, and how the system-wide resolver  is used to determine it. But [InetAddressResolver.lookupByAddress](https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/net/spi/InetAddressResolver.html#lookupByAddress(byte%5B%5D)) talks about host name, maybe it should be updated to use FQDN term?

`name` is missing here: `fully qualified domain` -> `fully qualified domain name`

Reply:
> Should we address that in this PR or a separate one?

I'm good with both, but having it as a separate PR might require an additional CSR if docs modifications alter the semantics:

> Q: If the text of the javadoc of a public exported API is changing, is a CSR request needed?
> A: A CSR request is required if the specification of a public exported API. Not all javadoc updates are specification changes. For example, typo fixes and rephrasings that do not alter the semantics of the API in question do not require CSR review.

[[CSR FAQ]](https://wiki.openjdk.org/display/csr/CSR+FAQs)

> Good catch. Fixed.

Thank you

Reply:
I spoke to Aleksei about the InetAddressResolver javadoc and whether it should be updated in this PR. We agreed that we will take it up separately. I've created https://bugs.openjdk.org/browse/JDK-8313947 to track that discussion.

The PR in the current state has all other reviews implemented and I've updated and finalized the CSR with the text that's in this PR.

Reply:
Thanks for creating a separate bug to track terms used in the `InetAddressResolver` and `InetAddress` classes. This PR looks good to me.